newdle/client/src/components/NewdleTitle.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Container, Label} from 'semantic-ui-react';
+import styles from './NewdleTitle.module.scss';
+
+export default function NewdleTitle({title, author, label}) {
+  return (
+    <Container text className={styles.box}>
+      <div className={styles.title}>
+        <h1 className={styles.header}>{title}</h1>
+        {label && (
+          <Label
+            color={label === 'finished' ? 'blue' : 'green'}
```

Comment:
please don't check human-targeted text for something like this - it'll break in the future with i18n. better to use props to specify the state of the newdle, and then based on this show the label (if it's not null) and determine its color.

if you want to keep it for now, change the prop type from string to `.oneOf(['finished', 'ongoing'])` so it's absolutely clear that this is not just a text that will be displayed